django_celery_beat/migrations/0011_fix_migration_history.py

```diff
@@ -10,22 +10,6 @@ class Migration(migrations.Migration):
     ]
 
     operations = [
-        # delete the squashed migration history of migrations replaced by
-        # 0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416
-        # the squashed file was convertd to a normal migration and the py files
-        # were deleted, so we must also delete them from the history.
-        # this is necessary because the squashed file was somehow screwing up
-        # this history for version 1.3.0 installs that were upgraded.
-        migrations.RunSQL([(
-            """
-            delete from django_migrations where app='django_celery_beat' and (
-            name = '0005_add_solarschedule_events_choices'
-            or name = '0006_auto_20180210_1226'
-            or name = '0006_auto_20180322_0932'
-            or name = '0007_auto_20180521_0826'
-            or name = '0008_auto_20180914_1922'
-            )
-            """, None)],
-            reverse_sql=migrations.RunSQL.noop
-        ),
+        # operations removed, migration kept to keep external
+        # reverse dependencies intact
```

Comment:
This migration can just be deleted, it never shipped as a version and has only existed for a few days.

Reply:
You could also update it to remove `0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416` so some don't have a ghost migration. Or keep `0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416` and have it be empty

Reply:
I only deleted `0005_add_solarschedule_events_choices_squashed_0009_merge_20181012_1416` altogether because I have not found any way to keep it: Not empty, not unmodified, nothing.

Reply:
Will delete `0011`, okay

Reply:
Resolving, please re-open as needed